assets/themes/default/_variables.scss

```diff
@@ -24,14 +24,14 @@ $margin: 0 $horizontal-margin;
 
 /* Fonts */
 
-$route159: "Route159", serif;
-$playfair: "Playfair Display", serif;
-$lora: "Lora", serif;
+$route159: "Vazir", "Route159", serif;
```

Comment:
This font is for the interface. Maybe you could also import the Shabnam font as suggested in #783 instead of Vizir for this one? This can be done in another PR, but if you don't want to do it in this one, could you at least restore Route159? It looks like there are some margin issues with this font on some buttons (but not in other places strangely).

![bildo](https://user-images.githubusercontent.com/16254623/85204173-e0c4d280-b312-11ea-8d19-b770e04a0841.png)

(it is quite subtle, but the text is not perfectly centered vertically)

Reply:
could we do what was suggested and only enable this for certain Unicode ranges?

Reply:
I believe using [non-Latin](https://github.com/rastikerdar/vazir-font/tree/master/dist/Farsi-Digits-Without-Latin) one would have the same effect. If we prioritize it, only Persian and Arabic chars will be effected.